airflow/migrations/versions/0041_3_0_0_add_uuid_primary_key_to_task_instance_.py

```diff
@@ -0,0 +1,273 @@
+                RAISE NOTICE 'pgcrypto extension could not be installed due to insufficient privileges; using fallback';
+                pgcrypto_installed := FALSE;
+            WHEN OTHERS THEN
+                RAISE NOTICE 'An unexpected error occurred while attempting to install pgcrypto; using fallback';
+                pgcrypto_installed := FALSE;
+        END;
+    END IF;
+END $$;
+
+CREATE OR REPLACE FUNCTION uuid_generate_v7(p_timestamp timestamp with time zone)
+RETURNS uuid
+LANGUAGE plpgsql
+PARALLEL SAFE
+AS $$
+DECLARE
+    unix_time_ms CONSTANT bytea NOT NULL DEFAULT substring(int8send((extract(epoch FROM p_timestamp) * 1000)::bigint) from 3);
+    buffer bytea;
+    pgcrypto_installed BOOLEAN := EXISTS (SELECT 1 FROM pg_extension WHERE extname = 'pgcrypto');
+BEGIN
+    -- Use pgcrypto if available, otherwise use the fallback
+    -- fallback from https://brandur.org/fragments/secure-bytes-without-pgcrypto
+    IF pgcrypto_installed THEN
+        buffer := unix_time_ms || gen_random_bytes(10);
+    ELSE
+        buffer := unix_time_ms || substring(uuid_send(gen_random_uuid()) FROM 1 FOR 5) ||
+                  substring(uuid_send(gen_random_uuid()) FROM 12 FOR 5);
+    END IF;
+
+    -- Set UUID version and variant bits
+    buffer := set_byte(buffer, 6, (b'0111' || get_byte(buffer, 6)::bit(4))::bit(8)::int);
+    buffer := set_byte(buffer, 8, (b'10'   || get_byte(buffer, 8)::bit(6))::bit(8)::int);
+    RETURN encode(buffer, 'hex')::uuid;
+END
+$$;
+"""
+
+pg_uuid7_fn_drop = """
+DROP FUNCTION IF EXISTS uuid_generate_v7(timestamp with time zone);
+"""
+
+# MySQL-specific UUID v7 function
+mysql_uuid7_fn = """
+DROP FUNCTION IF EXISTS uuid_generate_v7;
+CREATE FUNCTION uuid_generate_v7(p_timestamp DATETIME(3))
+RETURNS CHAR(36)
+DETERMINISTIC
+BEGIN
+    DECLARE unix_time_ms BIGINT;
+    DECLARE time_hex CHAR(12);
+    DECLARE rand_hex CHAR(24);
+    DECLARE uuid CHAR(36);
+
+    -- Convert the passed timestamp to milliseconds since epoch
+    SET unix_time_ms = UNIX_TIMESTAMP(p_timestamp) * 1000;
+    SET time_hex = LPAD(HEX(unix_time_ms), 12, '0');
+    SET rand_hex = CONCAT(
+        LPAD(HEX(FLOOR(RAND() * POW(2,32))), 8, '0'),
+        LPAD(HEX(FLOOR(RAND() * POW(2,32))), 8, '0')
+    );
+    SET rand_hex = CONCAT(SUBSTRING(rand_hex, 1, 4), '7', SUBSTRING(rand_hex, 6));
+    SET rand_hex = CONCAT(SUBSTRING(rand_hex, 1, 12), '8', SUBSTRING(rand_hex, 14));
+
+    SET uuid = LOWER(CONCAT(
+        SUBSTRING(time_hex, 1, 8), '-',
+        SUBSTRING(time_hex, 9, 4), '-',
+        SUBSTRING(rand_hex, 1, 4), '-',
+        SUBSTRING(rand_hex, 5, 4), '-',
+        SUBSTRING(rand_hex, 9)
+    ));
+
+    RETURN uuid;
+END;
+"""
+
+mysql_uuid7_fn_drop = """
+DROP FUNCTION IF EXISTS uuid_generate_v7;
+"""
+
+ti_table = "task_instance"
+
+# Foreign key columns from task_instance
+ti_fk_cols = ["dag_id", "task_id", "run_id", "map_index"]
+
+# Foreign key constraints from other tables to task_instance
+ti_fk_constrains = [
```

Comment:
```suggestion
ti_fk_constraints = [
```
nit

Reply:
Done in https://github.com/apache/airflow/pull/43243/commits/0f5dc0a0e496a38d53003ee9dc2941af0d8c9b74